R/openapi-types.R

```diff
@@ -70,6 +70,24 @@ add_api_info_onLoad <- function() {
     as.character,
     location = c("query", "path")
   )
+  addApiInfo(
+    "date-time",
+    c("POSIXct", "POSIXt"),
+    "[^/]+",
+    lubridate::as_datetime,
+    format = "date-time",
+    location = c("query", "path"),
+    realType = "string"
+  )
+  addApiInfo(
+    "date",
+    c("Date"),
+    "[^/]+",
+    lubridate::as_date,
+    format = "date",
```

Comment:
Is there a {base} method we can use? `{lubridate}` is currently not a direct dependency.

Reply:
I'll need to read the differences - https://lubridate.tidyverse.org/reference/as_date.html#:~:text=Called%20on%20a%20POSIXct%20object,%22UTC%22%20by%20default). - I think as.Date can work... but there are timezone gotchas (which are worse in base R I think)

Reply:
I agree it is 100% better using lubridate. If we need it, we can use `function(d) { lubridate::as_date(d) }` and have lubridate as a Suggests in the DESCRIPTION file.

Reply:
Done - with some protection/fallback if lubridate not available